### PR TITLE
Update netframework.md

### PR DESCRIPTION
**Pull request by iwasdavid**
I needed to manually add the `OpenTelemetry.Exporter.Console` package to get the example on this page to build successfully.

### CODE REVIEW

File: content/en/docs/instrumentation/net/netframework.md [hunk 1]
```diff
@@ -19,8 +19,9 @@ Initialization for ASP.NET is a little different than for ASP.NET Core.
 
 First, install the following NuGet packages:
 
-- [OpenTelemetry.Instrumentation.AspNet](https://www.nuget.org/packages/OpenTelemetry.Instrumentation.AspNet/)
+- [OpenTelemetry.Instrumentation.AspNet](https://www.nuget.org/packages/OpenTelemetry.Instrumentation.AspNet)
 - [OpenTelemetry.Extensions.Hosting](https://www.nuget.org/packages/OpenTelemetry.Extensions.Hosting)
```
Comment (cijothomas) [inline content/en/docs/instrumentation/net/netframework.md:23]: Extensions.Hosting wont be needed. Can you remove that too?

**Review by cijothomas — APPROVED** [review]

**Comment by cartermp** [comment]
@iwasdavid thanks! If you could address @cijothomas's comment then we should be good to go.

**Comment by cartermp** [comment]
ah boy I think I know what's up. @iwasdavid if you can comment:

`/fix:refcache`

then the action should pick up. I'll see what we can do orthogonally about making it work for anyone

**Review by cijothomas — APPROVED** [review]

**Comment by svrnm** [comment]
Ok.... there is something funky going on with the action, I'll raise an issue for that, in the meantime I try to fix your refcache issue locally

**Review by svrnm — APPROVED** [review]

**Pull request merged**